### ISSUE TITLE
HowTo

### ISSUE THREAD

**Issue by HenrikEckermann** [nteract/hydrogen#1145]
Hi, 
first of all, thanks for making this great tool!!

I want to replace working in RStudio with rmarkdown by working completely in Atom using Hydrogen. I need to write scientific articles and analyze data in R and Python. Is there a forum specifically for hydrogen/nteract because the following are more questions and not (yet) issues. I read the documentation of hydrogen but this is not covered there. What I would want to do is:

- Write Python&R code and scientific text (sometimes math-formulas with latex) in one file (currently rmarkdown) all in one document (at least R + markdown or Python + Markdown)
- Exporting this into docx, pdf and html must be possible with the option to hide code and only print code output in the output file
- would be nice if features of knitr are available: E.g. hide code in output file or hide just code output.

Basically, a workflow like in rmarkdown must be possible. Is this realistic or is this unrealistic at this stage of the development?

Question I have (please refer me to a forum if there exists a place to get support)
1. In Rstudio I can use keyboard short cuts to insert a code chunk like this:

\```{r}

\```

If I would use Atom do I have to type this manually for every code chunk or are there maybe shortcuts to create r/python chunks? Or could I use something else than ```to create a chunk in a multilanguage document?

**Comment by BenRussert** [comment]
You can make [snippets](http://flight-manual.atom.io/using-atom/sections/snippets/) and [keyboard shortcuts](http://flight-manual.atom.io/behind-atom/sections/keymaps-in-depth/) to your heart's content with atom.

**Comment by HenrikEckermann** [comment]
Hydrogen + pweave +snippets in Atom = **AWESOME!!!**
